pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -1411,21 +1422,26 @@ func (a *ApplicationRestoreController) skipVolumesFromRestoreList(
 			logrus.Infof("skipping namespace %s for restore", bkupVolInfo.Namespace)
 			continue
 		}
+		ns := val
+		var pvcName string // Declare the pvcName variable
+		if objects != nil {
+			// get corresponding pvc object from objects list
+			pvcObject, err := volume.GetPVCFromObjects(objects, bkupVolInfo)
+			if err != nil {
+				return newVolInfos, existingInfos, err
+			}
+			pvcName = pvcObject.Name
 
-		// get corresponding pvc object from objects list
-		pvcObject, err := volume.GetPVCFromObjects(objects, bkupVolInfo)
-		if err != nil {
-			return newVolInfos, existingInfos, err
+		} else {
```

Comment:
Can you please comment what is selected in case of object is nil? 
How is vInfo.PVC different from volume.GetPVCFromObjects(objects, bkupVolinfo).